web_server/social_distribution/templates/index.html

```diff
@@ -3,25 +3,59 @@
 <html>
 
 <head>
-    <title>Social Distribution</title>
+	<title>Social Distribution</title>
+	<script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
+        integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
+        crossorigin="anonymous"></script>
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
         integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+	<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
+		integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
+		crossorigin="anonymous"></script>
 </head>
 
 <body>
+	<nav class="navbar navbar-expand-lg navbar-light bg-light">
+		<a class="navbar-brand" href="/">
+			<img src="{% static '/users/images/logo.jpg' %}" width="30" height="30" class="d-inline-block align-top" alt="">
+			Mandala
+		</a>
+		<button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+			<span class="navbar-toggler-icon"></span>
+		</button>
+		
+		<div class="collapse navbar-collapse" id="navbarNav">
+			<ul class="navbar-nav mr-auto">
+				<li class="nav-item active">
+					<a class="nav-link" href="/posts/">Public</a>
+				</li>
+				<li class="nav-item active">
+					<a class="nav-link" href="/">Friends</a>
+				</li>
+				<li class="nav-item active">
+					<a class="nav-link" href="/">Request</a>
+				</li>
+				
+			</ul>
+			<div class="form-inline my-2 my-lg-0">
+				<ul class="navbar-nav">
+					<li class="nav-item">
+						<a class="nav-link" href="{% url 'profile' %}"><img src="{% static '/users/images/logo.jpg' %}" width="30" height="30" class="d-inline-block align-top" alt=""></a>
+					</li>
+					<li class="nav-item">
+						<a class="nav-link" href="{% url 'logout' %}">Logout</a>
+					</li>
+				</ul>
+			</div>
+		</div>
+	</nav>
+	
+    {% block content %}
+	{% endblock content %}
+	
+    
+    
 
-
-    {% block content %}{% endblock content %}
-
-    <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
```

Comment:
I might be out to lunch here, but if the scripts are moved to the head rather than the foot of the document, then if they aren't protected with a onready function call and they attempt to change the document on load, the document might not have rendered yet and the js function might fail. It's just something to be aware of.